modules/EnsEMBL/Web/SpeciesDefs.pm

```diff
@@ -1574,11 +1573,11 @@ sub species_label {
       ## Pan-compara species - get label from metadata db
       my $info = $self->get_config('MULTI', 'PAN_COMPARA_LOOKUP');
       if ($info) {
-        if ($info->{$key}) {
-          $label = $info->{$key}{'display_name'}
+        if ($info->{$url}) {
```

Comment:
is that right that we need the species URL as the key here. Before it was the production_name that was the key?

Reply:
Yes - see comments above.